File: Cursos/POO/Programación Orientada a Objetos POO/Code/PHP/UberBlack.php
<?php
require_once('Car.php');

class UberBlack extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
        // paent hace referencia a la clase padre
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}
?>

File: Cursos/POO/Programación Orientada a Objetos POO/Code/JAVA/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, ArrayList<String>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, Map<String, ArrayList<String>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        // Super hace referencia a la clase padre (Car)
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
        
    }
}
